climakitae/plotting.py

```diff
@@ -0,0 +1,35 @@
+import cartopy.crs as ccrs
+import hvplot.xarray
+import hvplot.pandas
+
```

Comment:
rather than create this 'plotting.py' I think I'd have put this in explore.py, as this is essentially a smaller component piece of what is involved in those larger panels.